_data/changelogs/pattern-user-profile-pronouns.yml

```diff
@@ -2,14 +2,22 @@ title: Pronouns
 type: pattern
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Added an archive.org link to the "The importance of personal pronouns" reference link.
+    summaryAdditional:
+    affectsGuidance: true
+    githubPr: 3088
+    githubRepo: uswds-site
   - date: 2025-01-23
     summary: De-linked dead hyperlink in references.
     summaryAdditional: De-linked pronouns reference link.
+    affectsGuidance: true
```

Comment:
> [!note]
> These were missing tags, but that might have been intentional. Please flag if they should be removed.

Reply:
I agree, and we'll need to be very clear about what exactly affects guidance as we move forward. These types of changes affect _content_ and link references but nothing affects _guidance_ until a point when we make a change to what we suggest teams consider or do

Reply:
That makes sense. I removed the `affectsGuidance` tag from these in [410a474](https://github.com/uswds/uswds-site/pull/3088/commits/410a47486a17a66fc0b055500d57618b7041ea28). 

My main concern is leaving the "affects" field empty. I like the idea of always giving users a quick view into what the change touches. Should we create a different tag that can label what this change affects?